torch/jit/frontend.py

```diff
@@ -142,7 +142,7 @@ def get_jit_class_def(cls, self_name):
     return build_class_def(ctx, py_ast.body[0], method_defs, self_name)
 
 
-def get_jit_def(fn, def_name, self_name=None):
+def get_jit_def(fn, def_name, self_name=None, unused=False):
```

Comment:
Instead of threading through another argument, can you just invoke `should_drop(fn)` in the body ?